SKILL.tsv

```diff
@@ -921,114 +921,114 @@ SKILL_20150317_000920	Thrust
 SKILL_20150317_000967	Critical + #{CaptionRatio}#{nl}Evasion - #{CaptionRatio2}#{nl}Duration #{CaptionTime}# seconds
 SKILL_20150317_000968	Synchro Thrusting
-SKILL_20150317_000969	{#DD5500}{ol}[Physical] - [Stab] - [Strike]{/}{/}{nl}Attack enemy using shield and spear.
+SKILL_20150317_000969	{#DD5500}{ol}[Physical] - [Stab] - [Strike]{/}{/}{nl} Inflict damage on an enemy using your shield and spear.
 SKILL_20150317_000970	Spear Attack #{SkillAtkAdd}#{nl}Shield Attack #{CaptionRatio}#{nl}Splash #{SkillSR}#
 SKILL_20150317_000971	Long Stride
-SKILL_20150317_000972	{#DD5500}{ol}[Physical] - [Stab]{/}{/}{nl}Teleport and attack enemies within range.
+SKILL_20150317_000972	{#DD5500}{ol}[Physical] - [Stab]{/}{/}{nl} Inflict damage to an enemy by jumping to your designated location.
 SKILL_20150317_000973	Spear Throw
 SKILL_20150317_000974	Shield Charge
-SKILL_20150317_000975	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl} Carry shield and run. Enemies hit nearby falls down.
+SKILL_20150317_000975	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl} Run while holding up your shield. The nearest target who is hit by the shield falls down.
 SKILL_20150317_000976	Attack #{SkillAtkAdd}#{nl}Max. Duration 10 seconds{nl}Consume STA
 SKILL_20150317_000977	Montano
-SKILL_20150317_000978	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Attack bottom of the target strongly.
+SKILL_20150317_000978	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Uses a strong attack on the lower part of a target.
 SKILL_20150317_000979	Attack #{SkillAtkAdd}#{nl}{#339999}{ol}[Stun]{/}{/} Duration 7 seconds
 SKILL_20150317_000980	Targe Smash
-SKILL_20150317_000981	{#DD5500}{ol}[Physical] [Strike]{/}{/}{nl} Whack strongly subject to the shield will increase. Target is stuck in a dark state with high probability.
+SKILL_20150317_000981	{#DD5500}{ol}[Physical] [Strike]{/}{/}{nl} Strongly smash your shield on the target to inflict damage. Grants a high chance for the target to fall into the Darkness state.
 SKILL_20150317_000982	Attack #{SkillAtkAdd}#{nl}{#339999}{ol}[Dark]{/}{/} Duration 3 seconds{nl}Splash #{SkillSR}#
 SKILL_20150317_000983	Shield Push
-SKILL_20150317_000984	{#DD5500}{ol}[Physical] [Strike]{/}{/}{nl}Interfere enemy with shield.{nl} Target falls under unbalanced state.
+SKILL_20150317_000984	{#DD5500}{ol}[Physical] [Strike]{/}{/}{nl}Use your shield to interfere with an enemy. {nl} The target will fall into the Unbalanced state.
 SKILL_20150317_000985	Attack #{SkillAtkAdd}#{nl}{#339999}{ol}[Unbalance]{/}{/} Duration 5 seconds
 SKILL_20150317_000986	Impaler
-SKILL_20150317_000987	{#DD5500}{ol}[Stab]{/}{/}{nl} I go to plug through the small monsters in the window. If you use the skill to take down to the ground after you plug in the window will increase.
+SKILL_20150317_000987	{#DD5500}{ol}[Stab]{/}{/}{nl} Impales a small monster with a spear. Can use this skill with a skewered monster to inflict damage.
```

Comment:
"Impales a small monster on your spear. Using this skill while a monster is skewered inflicts additional damage"? Need confirmation on skill mechanic, i.e. does the damage change depending on whether there is a monster impaled.